src/main/java/meditracker/medication/Medication.java

```diff
@@ -19,6 +19,25 @@ public class Medication {
     private String remarks;
     private String repeat;
 
+    /**
+     * Constructs a new Medication object with default placeholder values.
+     * Used by MedicationManager to populate medication data from the save file.
+     */
+    public Medication() {
+        double placeholderValue = -1.0;
+
+        setName("");
+        setQuantity(placeholderValue);
+        setDosage(placeholderValue);
+        setDosageMorning(placeholderValue);
+        setDosageAfternoon(placeholderValue);
+        setDosageEvening(placeholderValue);
+        setExpiryDate("");
+        setIntakeFreq("");
+        setRemarks("");
+        setRepeat("");
```

Comment:
Can you help me add a new field called "dayAdded", its an integer. This is to record the day the user has added the medication.

Reply:
Looks like it is related to #84. Will update after that PR has been merged in.